vhd/packer/install-dependencies.sh

```diff
@@ -422,6 +305,7 @@ pullContainerImage "docker" "busybox"
 echo "  - busybox" >> ${VHD_LOGS_FILEPATH}
 
 K8S_VERSIONS="
+1.18.0-alpha.5
```

Comment:
We haven't previously put pre-releases in the VHD. Is this for optimizing e2e tests?

Reply:
Yeah, I thought it would make sense to start doing this to enable us to level up our testing standards for pre-release versions.

Reply:
Should we do the same for windows?

Reply:
I vote yes!